tests/jerry-test-suite/es2015/25/25.04/25.04.05/25.04.05-001.js

```diff
@@ -1,16 +0,0 @@
-/* Copyright JS Foundation and other contributors, http://js.foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-assert (Promise.prototype.length === 1);
```

Comment:
Maybe he wanted to test Promise.length

Reply:
Standard(https://www.ecma-international.org/ecma-262/6.0/#sec-properties-of-the-promise-prototype-object) does not specify `length` property.
I checked this with other major browsers and all showed the same result.
Is this test intended or is there any special reason?

Reply:
I suspect he wanted to write `Promise.length` but he accidentally added prototype.

`assert (Promise.length === 1);` should be the correct assertion. If there is another test with this check, then it can be deleted.

Reply:
Fixed it to check `Promise.length`.